src/db/shoes.ts

```diff
@@ -0,0 +1,293 @@
+import { IPrototypeItem } from '../pages/templates/items';
+
+const shoes: IPrototypeItem[] = [
```

Comment:
If I see anywhere "IPtorotypeItem", I won't quess what it is.  But that's just my opinion